src/components/groups/group.service.ts

```diff
@@ -21,6 +25,7 @@ export const getGroup = async (req: Request, res: Response, next: NextFunction)
     .withGraphFetched('users')
 
   if (group) {
+    group.description = formatMdToSafeHTML(group.description)
     req.group = group
```

Comment:
Just another little thing you can do in these situations is using the object rest operator:
```
req.group = { ...group, description: formatMdToSafeHTML(group.description) }
```

It's perfectly fine as it is, it's just another way to do it.

Reply:
Spicy, gonna use it! Thanks!